tests/internal/log.c

```diff
@@ -0,0 +1,124 @@
+#include <fluent-bit/flb_sds.h>
+#include <fluent-bit/flb_mem.h>
+#include <fluent-bit/flb_log.h>
+
+#include "flb_tests_internal.h"
+
+#define TIMEOUT             5
+#define TEST_RECORD_01      "this is a test message"
+#define TEST_RECORD_01_SIZE sizeof(TEST_RECORD_01) - 1
+
+#define TEST_RECORD_02      "other type of message"
```

Comment:
ditto.

```diff
-#define TEST_RECORD_02      "other type of message"
+#define TEST_RECORD_02      "other type of message😵"
```

Reply:
Actually, the current implementation should be agnostic from that since it just compares bytes. Note: it uses 50% of the incoming string to assume is "similar":

- https://github.com/fluent/fluent-bit/blob/eduardo-log-cache/src/flb_log.c#L213-L228

Reply:
Hmm…, got it.